stdlib/2and3/keyword.pyi

```diff
@@ -0,0 +1,6 @@
+# Stubs for keyword
+
+from typing import Sequence
+
+def iskeyword(s: str) -> bool: ...
```

Comment:
This also works on `bytes`, `Tuple[str]`, `Tuple[bytes]`, `frozenset[str]`, etc.

We could use something like `Iterable[Union[str, bytes]]`, but it also needs to be hashable.
Maybe `Union[str, bytes]` is good enough.